### ISSUE TITLE
Add "Likes" section to profile

### ISSUE THREAD

**Issue by mxstbr** [withspectrum/spectrum#4657]
This should show threads one liked!

**Comment by ThomasRoest** [comment]
This should be added as a new tab here?

<img width="761" alt="image 2019-02-18 at 10 07 02 am" src="https://user-images.githubusercontent.com/4428811/52939648-0465ba80-3365-11e9-9ccd-55e5d8e76bc3.png">

**Comment by mxstbr** [comment]
Yes exactly!

**Comment by brianlovin** [comment]
How do we want to split up thread likes vs comment likes? Do all of those appear here? Or is this only for thread likes (this is my assumption)

**Comment by mxstbr** [comment]
I think a good v1 would be only thread likes, I am not sure how to handle message likes right now.

**Comment by ThomasRoest** [comment]
where are these likes ( for threads ) currently stored? which table?

**Comment by brianlovin** [comment]
@ThomasRoest in the `reactions` table.

**Comment by brianlovin** [comment]
Agreed though that a v1 for showing thread likes seems like the highest impact work

**Comment by ThomasRoest** [comment]
just some notes from exploring this

quering the db with `r.db('spectrum').table('reactions').filter({ userId: '1'})` results in 

```
{
"id":  "1014bbbf-00b8-4b91-840b-bc1005f2d4cd" ,
"messageId":  "12727fcb-0c20-42ba-86cf-c2094b37f78e" ,
"timestamp": Thu Nov 22 2018 06:14:53 GMT+00:00 ,
"type":  "like" ,
"userId":  "1"
}

```
These include both comment and thread likes?

Compent for profile 

`<SegmentedControl>` in `src/views/user/index.js`


current reaction model
`spectrum/analytics/models/reaction.js`

**Comment by ThomasRoest** [comment]
so probably something like this?

`r.db('spectrum').table('threadReactions').filter({ userId: currentUserId})` to get the threads?

current threadReaction model
`api/models/threadReaction.js`

**Comment by ThomasRoest** [comment]
so maybe something like this to get the threads / threads id's ?`r.db('spectrum').table('threadReactions').filter({ userId: currentUserId})`

**Comment by mxstbr** [comment]
You should use an index, otherwise it is going to be _really_ slow!

```JS
db.table('threadReactions').getAll(currentUserId, { index: 'userId' })
```

If that index does not exist yet you would need to create a migration script with `yarn run rethinkdb:migrate create thread-reactions-userid-index`.

**Comment by brianlovin** [comment]
the `threadReactions` table does have a `userId` index, so @mxstbr's last comment should work @ThomasRoest :)

**Comment by ThomasRoest** [comment]
So currently trying to figure out how data fetching works. Looking at the data being fetched in the user profile. Can you explain what the 'threadConnection` represents? and the difference with 'threads`, as shown in the example?

<img width="665" alt="threadconnection" src="https://user-images.githubusercontent.com/4428811/53325255-2fab5500-38e3-11e9-9bdc-e21f3bbea5c7.png">

**Comment by brianlovin** [comment]
I think the `threads` prop there is provided to make sure that every `ThreadFeed` component has some consistent prop consumption, regardless of the wrapping query. Eg. getting a user's threads returns a prop `data.user.threadConnection` and getting a community's threads returns a prop `data.community.threadConnection` - this would be annoying to handle every case. So instead, we just make sure that there will always be a `data.threads` - you can see where this gets set in `shared/graphql/queries/user/getUserThreadConnection.js:66`

Let me know if that makes sense!

**Comment by ThomasRoest** [comment]
From what I've found out the thread is created based on the `kind: 'creator' | 'participant'` so there should be an addition here,  for example 'liked' and added to 

`/api/queries/user/threadConnection.js`

and 

`/api/models/thread.js'

**Comment by brianlovin** [comment]
Looks correct to me @ThomasRoest :)
